src/_micro-api/libraries/signalDecoder/src/signalDecoder.cpp

```diff
@@ -1109,23 +1153,24 @@ const bool ManchesterpatternDecoder::doDecode() {
 				   //if (i < pdec->messageLen-minbitlen)
 				if (ManchesterBits.valcount < minbitlen)
 				{
-					if (isShort(pdec->message[i]) && i < pdec->messageLen - 1 && !isShort(pdec->message[i + 1])) {
-						// unequal number of short pulses. Restart, but one pulse ahead i is incremented at end of while loop
-						i = pdec->mstart;
-					}
+//					if (isShort(pdec->message[i]) && i < pdec->messageLen - 1 && !isShort(pdec->message[i + 1])) {
```

Comment:
Diesen Teil hatte ich drinnen, damit man sich neu resyncen kann, wenn man z,B. mitten in der Übertragung erst empfängt.